lib/lotus/model/adapters/sql/command.rb

```diff
@@ -55,10 +59,41 @@ def update(entity)
           # @api private
           # @since 0.1.0
           def delete
-            @collection.delete
+            _rescue_database_error do
+              @collection.delete
+            end
           end
 
           alias_method :clear, :delete
+
+          private
+
+          # Rescues Sequel::DatabaseError in yielded block
+          #
+          # @api private
+          def _rescue_database_error
```

Comment:
Now that the rescue has been simplified to raising one error, I'll just define a rescue for each of the public methods.